zzz_generated/all-clusters-app/zap-generated/cluster-objects.h

```diff
@@ -0,0 +1,1305 @@
+/*
```

Comment:
We don't need to generate a unique `cluster-objects.cpp`, and `.h` file for each app. In fact, the whole goal of this cluster object model is to have just 1 version across all apps :) 

That version should be in `zzz_generated/common/zap-generated/`

Reply:
Done, thanks